File: Fronted/hw_17/4.js
// Задание 5: Факториал числа


function factorial(num){
    if (num <= 0){
        console.log('Enter positive number!')
    }
    else{
        let result = 1;
        let i = 1;
        while( i <= num){
            result *= i;
            i++;
        }
        console.log(result)
    }
    
}

let num = 5;
factorial(num);

File: Python basics/lesson3/4.py
n = int(input('Enter number: '))
factorial = 1

#soluton with for loop
for i in range (1, n+1):
    factorial *= i

print(factorial)


#soluton with while 
i=1
while i <= n:
    factorial *= i
    i += 1

print(factorial)